forum_amo/models.py

```diff
@@ -0,0 +1,17 @@
+"""
+Models para o app forum_app
+"""
+
+from django.db import models
+
+class Duvida(models.Model):
+
+    """Modelo para dúvidas"""
+
+    titulo = models.CharField(max_length=200)
+    descricao = models.TextField(max_length=550)
+    privacidade_autor = models.BooleanField(default=True)
```

Comment:
Pode tirar essa questão da privacidade, por enquanto. (E do *serializer* também.)

Mas a ideia ta certa, é um campo booliano, e depois tratamos de não enviar os dados do autor se for privado.